scipy/io/netcdf.py

```diff
@@ -987,6 +983,50 @@ def __setitem__(self, index, data):
                 self.data.resize(shape)
         self.data[index] = data
 
+    def _get_missing_value(self):
+        """
+        Returns the value denoting "no data" for this variable.
+
+        If this variable does not have a missing/fill value, returns None.
```

Comment:
Probably a good idea to include your comment on the PR here:

    If both _FillValue and missing_value are given, give precedence to _FillValue.
    The netCDF standard gives special meaning to _FillValue; missing_value is 
    just used for compatibility with old datasets.